### ISSUE TITLE
A Big Thanks!

### ISSUE THREAD

**Issue by twashing** [jafingerhut/jafingerhut.github.com#10]
Not an issue really. Just dropped in to say that the Clojure Cheatsheet with TipTip is my goto documentation. 

So a big thanks to all contributors. And let me / us know if there's any way to support your work.

Cheers
Tim

**Comment by jafingerhut** [comment]
Glad you like it!

It is fairly low maintenance work these days, with occasional bursts of additions when a new release of Clojure comes out.

I know it does not contain everything included with Clojure.  I could imagine adding a few more things to it that aren't there already, but it isn't clear to me what the most important things are that are left out.

**Comment by twashing** [comment]
Right right. Well I'll keep my eyes peeled as to any docs that could be
added.

Thanks again!

Tim Washington
Interruptsoftware.com <http://interruptsoftware.com>
415.283.8889

**Comment by jafingerhut** [comment]
Here is a list of Clojure Vars (functions, macros, and others) that are in the namespaces used by the cheat sheet generator program, but not currently included in the cheat sheet.  I would not want to include them all, since some are fairly esoteric in their use, but I would not be surprised if some of them would be useful to add: https://github.com/jafingerhut/clojure-cheatsheets/blob/master/src/clj-jvm/unused-symbols.txt

**Comment by twashing** [comment]
Oh, very interesting. Things like clojure.core.async.impl.protocols/closed?
Available but not meant to be publicly consumed?

I could see that being very useful. Maybe under a *Miscellaneous* section,
or something.

Tim Washington
Interruptsoftware.com <http://interruptsoftware.com>
415.283.8889

**Comment by jafingerhut** [comment]
There is little or nothing about the core.async library in the cheat sheet right now.  In general, if something is there, it is good if it can have some kind of at least rough categorization of it, and yes, implementation details are best not included, I think.

**Comment by twashing** [comment]
Yes, agreed. Right now, Clojuredocs has my goto documentation for core.async
.

   - https://clojuredocs.org/clojure.core.async


Tim Washington
Interruptsoftware.com <http://interruptsoftware.com>
415.283.8889

**Comment by lkdm** [comment]
I don't even write Clojure, but I have to say I am really impressed with the UI for this documentation. Well done!

**Comment by jfingerh** [comment]
Thanks.  Some of the contributors to the project are as responsible for the UI as I am, but glad to hear you like it.  The code that produced it is all included and open source, so should not be difficult to reproduce for other documentation projects if you are interested.
